tutorials/apple-mdm-jamf-setup-guide.mdx

```diff
@@ -205,6 +98,20 @@ Here's one approach:
 - Re-add your test device to the production profile scope
 - Finally, remove the cloned profile
 
+### Running your own RADIUS server?
+
+If you run your own RADIUS server, you'll need to modify the Configuration Profile you uploaded to Jamf from Smallstep.
+Change the Certificate Trust settings for your `Wi-Fi` Payload so that they use your RADIUS server's Root CA Certificate instead of Smallstep's.
+You may need to add an additional `Certificate` payload for your RADIUS server.
```

Comment:
The "uploaded to Jamf from Smallstep" reads a bit weird

```suggestion
If you run your own RADIUS server, you'll need to modify the Configuration Profile you uploaded to Jamf from Smallstep.
Change the Certificate Trust settings for your `Wi-Fi` Payload so that they use your RADIUS server's Root CA certificate instead of Smallstep's.
You may need to add an additional `Certificate` payload for your RADIUS server.
```